examples/optuna_rapids.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+"""
+Optuna example using RAPIDS library for optimization.
+
+In this example, we perform HPO on Iris dataset using cuML's
+RandomForestClassifier. This should be used a starting point
+to extend the search for larger problems, and wider depths.
+
+To run this example:
+
+    $ python optuna_rapids.py
+
+Learn more about rapids: https://rapids.ai/
+"""
+import cudf
+import optuna
+from cuml.ensemble import RandomForestClassifier
+from cuml.metrics import accuracy_score
+from cuml.preprocessing.model_selection import train_test_split
+from sklearn.datasets import load_iris
+
+
+def train_and_eval(X_param, y_param, max_depth=16, n_estimators=100):
+    X_train, X_valid, y_train, y_valid = train_test_split(X_param,
+                                                          y_param,
+                                                          random_state=77)
+    classifier = RandomForestClassifier(max_depth=max_depth,
+                                        n_estimators=n_estimators)
+    classifier.fit(X_train, y_train)
+    y_pred = classifier.predict(X_valid)
+    score = accuracy_score(y_valid, y_pred)
+    return score
+
+
+def objective(trial, X_param, y_param):
+    max_depth = trial.suggest_int("max_depth", 7, 15)
+    n_estimators = trial.suggest_int("n_estimators", 100, 1000)
+    score = train_and_eval(X_param, y_param, max_depth=max_depth,
+                           n_estimators=n_estimators)
+    return score
+
+
+if __name__ == "__main__":
+    N_TRIALS = 10
+    study_name = "optuna_experiment"
+    storage_name = "sqlite:///simple_storage.db"
```

Comment:
[question] Can we run this example without an RDB storage?

Reply:
Yes, we can. Do you want me to change that?